fuzzing_tc/pool_launch/launcher.py

```diff
@@ -24,6 +25,8 @@ def __init__(self, command, pool_name):
         self.command = command.copy()
         self.environment = os.environ.copy()
         self.pool_name = pool_name
+        self.in_taskcluster = False
```

Comment:
A more generic way would be to do :

```python
self.in_taskcluster = "TASK_ID" in os.environ and "TASKCLUSTER_ROOT_URL" in os.environ
```

No need to update then in `load_params`